lib/matplotlib/backends/backend_ps.py

```diff
@@ -828,13 +825,13 @@ def draw_mathtext(self, gc, x, y, s, prop, angle):
             f"{x:g} {y:g} translate\n"
             f"{angle:g} rotate\n")
         lastfont = None
-        for font, fontsize, num, ox, oy in glyphs:
-            self._character_tracker.track_glyph(font, num)
+        for font, fontsize, ccode, glyph_index, ox, oy in glyphs:
+            self._character_tracker.track_glyph(font, glyph_index)
```

Comment:
Ditto use `text.index`, also adjust below.

Reply:
Not sure what you mean here; this isn't a DVI page?

Reply:
Ah yes sorry, got confused.